### PR TITLE
Feature/hall of fame

### PR DESCRIPTION
**Pull request by Jonas-Wendland**

### CODE REVIEW

File: pages/HallOfFame/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  function sortedDeceasedPets(data) {
+    let arrayCopy = [...data];
+    arrayCopy.sort(
+      (a, b) => b.lastUpdated - b.createdAt - (a.lastUpdated - a.createdAt)
+    );
+    return arrayCopy.slice(0, 100);
+  }
+
+  const top100 = sortedDeceasedPets(data);
```
Comment (doemser) [inline pages/HallOfFame/index.js:25]:
```suggestion
  const top100 = data.toSorted(
      (a, b) => b.lastUpdated - b.createdAt - (a.lastUpdated - a.createdAt)
    );
```

File: pages/HallOfFame/index.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  return (
+    <StyledBackground>
+      <div>
+        <StyledLink href={"/"}> ←</StyledLink>
+        <StyledHeading>Hall of Fame</StyledHeading>
+      </div>
+      <StyledList>
+        {top100Index.map((pet) => (
+          <StyledListItems key={pet._id}>
+            <p></p>
```
Comment (nichoop) [inline pages/HallOfFame/index.js:40]: Did you need that or you want to put something after?

File: pages/api/pets/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      const petData = request.body;
+      await Pet.create(petData);
+      // const pet = new Pet(petData);
+      // await pet.save();
+      response.status(201).json({ status: "Pet Created!" });
+    } catch (error) {
+      console.error(error);
+      response.status(400).json({ error: error.message });
+    }
+  }
```
Comment (nichoop) [inline pages/api/pets/index.js:28]: Just a curiosity, don't you also need PUT/PATCH to update?

Comment (Jonas-Wendland) [reply pages/api/pets/index.js:28]: no because we never update the objects they will be like they are

File: pages/HallOfFame/index.js (hunk 2 above)
Comment (Jonas-Wendland) [reply pages/HallOfFame/index.js:40]: you are right we dont need that we will change that later

**Review by nichoop — APPROVED** [review]
Nice, the database has been well implemented, everything is there.

**Pull request merged**